Robust.Server/GameStates/PVSSystem.cs

```diff
@@ -279,6 +279,10 @@ private void OnMapCreated(object? sender, MapEventArgs e)
         var transformQuery = EntityManager.GetEntityQuery<TransformComponent>();
         var viewerEntities = new EntityUid[sessions.Length][];
 
+        // Keep track of the index of each chunk we use for a faster index lookup.
+        var mapIndices = new Dictionary<uint, Dictionary<MapChunkLocation, int>>(4);
+        var gridIndices = new Dictionary<uint, Dictionary<GridChunkLocation, int>>(4);
```

Comment:
I figured it avoids the interface shenanigans, plus with the outer dictionary the lookups should be faster (as we just need to do the outer vismask one once per viewer)